app/controllers/idv/personal_key_controller.rb

```diff
@@ -18,6 +18,11 @@ class PersonalKeyController < ApplicationController
     skip_before_action :handle_fraud
 
     def show
+      if pii_is_missing
```

Comment:
Can we tweak this slightly so the `idv_personal_key_visited` analytics event below gets logged, but with an extra `pii_missing` property? That way we can keep tracking how often we are seeing the underlying issue.

Reply:
Tweaked.